{{cookiecutter.project_folder}}/.gitignore

```diff
@@ -4,15 +4,15 @@ __pycache__/
 *$py.class
 
 # C extensions
-*.so
+# *.so
```

Comment:
why commented?

Reply:
same above, should I simply uncomment these two?